src/Microsoft.VisualStudio.Composition/ReflectionHelpers.cs

```diff
@@ -219,6 +233,34 @@ internal static bool IsStatic(this MemberInfo exportingMember)
             throw new NotSupportedException();
         }
 
+        internal static bool IsStatic(this MemberRef exportingMemberRef)
```

Comment:
>IsStatic(this MemberRef exportingMemberRef) [](start = 29, length = 43)

Why isn't this a property on `MemberRef` itself? Which derived type are you *not* checking in this method? It seems that every member might be static (except constructors, but there isn't a ConstructorRef type, I think). If you promote the property to the base type, you can remove this method.

Reply:
Now that you've promoted the `IsStatic` property to the base type, we can remove this method, right?

---
In reply to: [264505835](https://github.com/Microsoft/vs-mef/pull/111#discussion_r264505835) [](ancestors = 264505835)